feat: allow users to specify custom migrations dir location (#8844)
confluentinc/ksql: docs/reference/migrations-tool-configuration.md

The password that will be used to connect to the ksqlDB server. A 
[username](#ksqlauthbasicusername) and password will be passed as part of 
HTTP basic authentication. 

Migrations Directory Configs
----------------------------

> ### `ksql.migrations.dir.override`

Reviewer: We should specify that the migration tool should be on version 0.25+

Author: Good call. I think this is shipping with 0.26 so that's what I put in the docs. I'll update separately if the 0.25 cut gets delayed and this manages to get picked up.

An optional config that allows you to specify the path to the directory
containing migrations files to be applied. This config is not needed if you
set up your migrations project using the `ksql-migrations new-project` command.

> If no override is provided, the migrations directory will be inferred relative 

Reviewer 2: ```suggestion
If no override is provided, the migrations directory is inferred relative 
```

to the migrations configuration file passed when using the `ksql-migrations` tool. 

> Specifically, the migrations directory will be inferred as a directory with name 

Reviewer 2: ```suggestion
Specifically, the migrations directory is inferred as a directory with name 
```
